changelog/next/changes/4689--null-comparision-ops.md

```diff
@@ -0,0 +1,2 @@
+We made [Three-Valued Logic](https://en.wikipedia.org/wiki/Three-valued_logic) comparisions possible for `null` and
+`bool` types. Additionally, these no longer emit a diagnostic when used.
```

Comment:
The three-valued logic was also there before, and typically you don't need to reference external resources in a changelog. Below is another suggestion, which also spells out the change in behavior more clearly. Finally, please stick to the maximum line length when writing changelogs.
```suggestion
The boolean operators `and`/`or` now work correctly for the type `null`.
Previously, `null and false` evaluated to `null`, and a warning was emitted.
Now, it evaluates to `false` without a warning.
```

Reply:
Fixed, thanks!